From 41968cb50b19e54b4e2fa5d451cee42a957464da Mon Sep 17 00:00:00 2001
Message-ID: <41968cb50b19e54b4e2fa5d451cee42a957464da.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 03:11:28 +0100
Subject: [PATCH v3 07/66] gnu: go-github-com-chris-ramon-douceur: Move to (gnu
 packages golang-web).

* gnu/packages/golang.scm (go-github-com-chris-ramon-douceur): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I9d0fdccc4ff557870dad60e9ac062ccb2e995979
---
 gnu/packages/golang-web.scm | 30 +++++++++++++++++++++++++++++-
 gnu/packages/golang.scm     | 28 ----------------------------
 2 files changed, 29 insertions(+), 29 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 8d8b4fb85c..79a69bc842 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -2,7 +2,7 @@
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
 ;;; Copyright © 2020 Martin Becze <mjbecze@riseup.net>
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
-;;; Copyright © 2021 Efraim Flashner <efraim@flashner.co.il>
+;;; Copyright © 2021, 2022 Efraim Flashner <efraim@flashner.co.il>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
 ;;; Copyright © 2022 Giacomo Leidi <goodoldpaul@autistici.org>
 ;;;
@@ -88,6 +88,34 @@ (define-public go-github-com-aymerick-douceur
     (description "This package provides a CSS parser and inliner.")
     (license license:expat)))
 
+(define-public go-github-com-chris-ramon-douceur
+  (package
+    (name "go-github-com-chris-ramon-douceur")
+    (version "0.2.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/chris-ramon/douceur")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1hfysznib0fqbp8vqxpk0xiggpp0ayk2bsddi36vbg6f8zq5f81n"))))
+    (build-system go-build-system)
+    (arguments
+     `(#:import-path "github.com/chris-ramon/douceur"))
+    (propagated-inputs
+     (list go-github-com-aymerick-douceur
+           go-github-com-gorilla-css))
+    (native-inputs
+     (list go-github-com-puerkitobio-goquery
+           go-github-com-andybalholm-cascadia
+           go-golang-org-x-net))
+    (home-page "https://github.com/chris-ramon/douceur/")
+    (synopsis "CSS parser and inliner")
+    (description "This package provides a CSS parser and inliner.")
+    (license license:expat)))
+
 (define-public go-github-com-gorilla-css
   (package
     (name "go-github-com-gorilla-css")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 7868f71206..a4f13b8c9d 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9357,34 +9357,6 @@ (define-public go-github-com-jmespath-go-jmespath
 transforms one JSON document into another through a JMESPath expression.")
     (license license:asl2.0)))
 
-(define-public go-github-com-chris-ramon-douceur
-  (package
-    (name "go-github-com-chris-ramon-douceur")
-    (version "0.2.0")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/chris-ramon/douceur")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "1hfysznib0fqbp8vqxpk0xiggpp0ayk2bsddi36vbg6f8zq5f81n"))))
-    (build-system go-build-system)
-    (arguments
-     `(#:import-path "github.com/chris-ramon/douceur"))
-    (propagated-inputs
-     (list go-github-com-aymerick-douceur
-           go-github-com-gorilla-css))
-    (native-inputs
-     (list go-github-com-puerkitobio-goquery
-           go-github-com-andybalholm-cascadia
-           go-golang-org-x-net))
-    (home-page "https://github.com/chris-ramon/douceur/")
-    (synopsis "CSS parser and inliner")
-    (description "This package provides a CSS parser and inliner.")
-    (license license:expat)))
-
 (define-public go-github-com-microcosm-cc-bluemonday
   (package
     (name "go-github-com-microcosm-cc-bluemonday")
-- 
2.41.0

